src/tests/utils/constants.cairo

```diff
@@ -75,3 +76,11 @@ fn DATA(success: bool) -> Span<felt252> {
     }
     data.span()
 }
+
+fn TRANSACTION_VERSION() -> felt252 {
+    utils::TRANSACTION_VERSION.try_into().unwrap()
+}
```

Comment:
```suggestion
const TRANSACTION_VERSION: felt252 = utils::TRANSACTION_VERSION.try_into().unwrap();
```
does this work? it'd make it easier to use

Reply:
No, unfortunately: `error: Function call is not supported outside of functions`

Reply:
To avoid using functions, I just redefined the versions as felts in the constants module